scripts/battery.sh

```diff
@@ -2,6 +2,11 @@
 # setting the locale, some users have issues with different locales, this forces the correct one
 export LC_ALL=en_US.UTF-8
 
+current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $current_dir/utils.sh
+
+IFS=' ' read -r -a show_charging_time <<< $(get_tmux_option "@dracula-battery-show-charging-time" "false")
```

Comment:
Put this into dracula.tmux

Reply:
Done (sorry for almost a year, but I have no time before).